engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/BucketedPartitionedUpdateByManager.java

```diff
@@ -49,6 +49,7 @@ protected BucketedPartitionedUpdateByManager(
             @NotNull final UpdateByWindow[] windows,
             @NotNull final ColumnSource<?>[] inputSources,
             @NotNull final QueryTable source,
+            final String[] persistentColumns,
```

Comment:
`preserved` is probably more in keeping with terminology from `update`. Is this `@NotNull`?